packages/lit-element/src/lib/updating-element.ts

```diff
@@ -396,9 +396,9 @@ export abstract class UpdatingElement extends HTMLElement {
     }
     this[finalized] = true;
     // finalize any superclasses
-    const superProto = Object.getPrototypeOf(this);
+    const superProto = Object.getPrototypeOf(this) as typeof UpdatingElement;
```

Comment:
what made this cast necessary?

Reply:
Huh. I guess https://github.com/Microsoft/TypeScript/issues/22875 is the latest on this. Doesn't seem like TS will track prototype type information...